webapp/src/hooks/useForm.ts

```diff
@@ -108,7 +108,7 @@ function useForm(initialFormFields: Record<FormFieldNames, ModalFormFieldConfig>
     };
 
     // Set value for a specific form field
-    const setSpecificFieldValue = (modifiedFormFields: Partial<Record<FormFieldNames, string>>) => {
+    const setSpecificFieldValue = (modifiedFormFields:Partial<Record<FormFieldNames, string>>) => {
```

Comment:
Revert this change. Fix lint.

Reply:
reverted these type of change may came due to sync, also eslint is not giving any error for this